docs/DeveloperGuide.md

```diff
@@ -505,6 +505,27 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Steps 1a1 to 1a2 are repeated until the data entered are correct.  
       Use case resumes from Step 2.
 
+**Use Case: UC08 - Find lessons with a specific name keyword**
+
+**MSS**:
+
+1. Tutor keys in a keyword in the required field to find lessons whose names contain this keyword.
```

Comment:
Find lessons with student names that contain this keyword?

Reply:
Fixed